lib/routers/profile/person.js

```diff
@@ -86,12 +92,6 @@ const getSingleProfile = req => {
       }
       return profile;
     })
-    .then(profile => {
-      if (req.establishment) {
-        profile.establishments = profile.establishments.filter(e => e.id === req.establishment.id);
-      }
-      return profile;
-    })
```

Comment:
This appears to be doing the same thing as scopeToEstablishment https://github.com/UKHomeOffice/asl-schema/blob/master/schema/base-model.js#L29-L40 so might be redundant?

Reply:
I'm not totally convinced by "appears to be". Could we add a unit test that makes sure it is doing it?